Updated Italian GUI configuration

troccolo opened the pull request:
added confirm setting as in English and German

CurlyMoo commented:
So what changed? The previous commit wasn't very verbose. I always assume it are just grammar fixes and not new information when not explicitly mentioned.

troccolo commented:
I updated Italian configuration GUI:
added translation of

confirm
Should the GUI ask for confirmation before switching the device? Default is 0 

confirm
Deve essere richiesta conferma prima di commutare il dispositivo? Il valore predefinito è 0 (non chiedere conferma).

pilino1234 commented:
Yes, I was testing some things and found that there was nothing in the manual about the confirm option in the GUI configuration document. 
I thought "[…]Add information about confirm setting." was verbose enough :smiley:
